src/operator/numpy/np_cumsum-inl.h

```diff
@@ -98,11 +98,11 @@ void CumsumForwardImpl(const OpContext& ctx,
   }
 
   Stream<xpu> *s = ctx.get_stream<xpu>();
-  MSHADOW_TYPE_SWITCH_WITH_BOOL(in.type_flag_, IType, {
+  MSHADOW_TYPE_SWITCH_WITH_BOOL(in.type_flag_, index_t, {
```

Comment:
I think you will not need this switch macro, right?

Reply:
Great point, reverted to IType.